zetaclient/ethrpc/types.go

```diff
@@ -0,0 +1,162 @@
+package ethrpc
```

Comment:
Can we consider moving these types to the common folder? 
In case we plan to build other debug tools which would use these types as well. 

Does not have to be done in this PR . just adding an issue would be enough

Reply:
let me make a comment for borrowed file
https://github.com/onrik/ethrpc/blob/main/types.go

Reply:
It's a temp solution and be used by zetaclient only

Reply:
the [commit](https://github.com/zeta-chain/node/pull/1665/commits/003e851e10e5f76fee146ac76cc4bb564b6c523b)